packages/@aws-cdk/custom-resources/test/provider-framework/integ.provider.expected.json

```diff
@@ -1052,18 +1052,7 @@
               "Action": "sts:AssumeRole",
               "Effect": "Allow",
               "Principal": {
-                "Service": {
-                  "Fn::Join": [
-                    "",
-                    [
-                      "states.",
-                      {
-                        "Ref": "AWS::Region"
-                      },
-                      ".amazonaws.com"
-                    ]
-                  ]
-                }
+                "Service": "states.test-region.amazonaws.com"
```

Comment:
Looks like a lot of our integration tests might be out of date. Can we update this separately? It unrelated to the lambda change.

Reply:
It is unrelated. I'll open a PR to address these separately.

Reply:
I ran `yarn integ --dry-run` on all integ tests, and this is what i got. i don't think there's a way for the integ tests to be out of date, as they get checked with `expected.json` on every build. So these integ changes have something to do with the current PR.